Resources/views/Collector/db.html.twig

```diff
@@ -359,10 +359,15 @@
                     </tr>
                     </thead>
                     <tbody>
-                    {% for class in classes %}
+                    {% for class in classes|keys %}
                         {% set contains_errors = collector.mappingErrors[manager] is defined and collector.mappingErrors[manager][class] is defined %}
                         <tr class="{{ contains_errors ? 'status-error' }}">
-                            <td>{{ class }}</td>
+                            <td>
+                                {%- set link = classes[class].file|file_link(classes[class].line) %}
+                                {%- if link %}<a href="{{ link }}" title="{{ class }}">{% else %}<span title="{{ class }}">{% endif %}
```

Comment:
From brief look, if you use `format_file` instead of `file_link`, looks like we wouldn't have to do this if checking ourselves

Reply:
That will show `App\Entity\Hello at line 9`. Or we drop the line but then clicking the link will not send you to the start of the class. Which do you prefer?

Reply:
Yeah let's remove all conditionals and leave in just 
```twig
<a href="{{ class.file|file_link(class.line) }}">{{class.class}}</a>
```

(I also couldn't find a purpose for that `<span>`)

We do exactly this on other place and nobody has complained yet.

Reply:
Yup the class will always be there anyway. Done.